g2o/types/slam2d/edge_pointxy.h

```diff
@@ -74,7 +74,7 @@ namespace g2o {
       }
 
 
-      virtual number_t initialEstimatePossible(const OptimizableGraph::VertexSet& , OptimizableGraph::Vertex* ) { return 0.;}
```

Comment:
I believe this was done to fix a warning for the int to float conversion. There's a `number_t cst(T);` function that does the proper cast in one of the helper headers to define constant number_t values.